api.js

```diff
@@ -3,11 +3,13 @@ const updateStore = require('./lib/sync/update-store');
 const updateStorageManager = require('./lib/middleware/update-storage-manager');
 const modelActions = require('./lib/actions/model');
 const collectionActions = require('./lib/actions/collection');
+const stopUpdatingStore = require('./lib/sync/stop-updating-store');
 
 module.exports = {
   reducer,
   updateStore,
   updateStorageManager,
   modelActions,
   collectionActions,
+  stopUpdatingStore,
```

Comment:
eh, `stopUpdateStore` doesn't really make sense as a command/sentence - I feel like this is still clear and also sounds good?